command/transfer.go

```diff
@@ -1,10 +1,47 @@
 package command
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/rubixchain/rubixgoplatform/core/model"
 )
 
 func (cmd *Command) TransferRBT() {
+	if cmd.senderAddr == "" {
+		cmd.log.Info("Sender address cannot be empty")
+		fmt.Print("Enter Sender DID : ")
+		_, err := fmt.Scan(&cmd.senderAddr)
+		if err != nil {
+			cmd.log.Error("Failed to get Sender DID")
+			return
+		}
+	}
+	if cmd.receiverAddr == "" {
+		cmd.log.Info("Receiver address cannot be empty")
+		fmt.Print("Enter Receiver DID : ")
+		_, err := fmt.Scan(&cmd.receiverAddr)
+		if err != nil {
+			cmd.log.Error("Failed to get Receiver DID")
+			return
+		}
+	}
+	if strings.Contains(cmd.senderAddr, ".") || strings.Contains(cmd.receiverAddr, ".") {
+		cmd.log.Error("Invalid sender or receiver address. Please provide valid DID")
+		return
+	}
+	if !strings.HasPrefix(cmd.senderAddr, "bafybmi") || len(cmd.senderAddr) < 59 || !strings.HasPrefix(cmd.receiverAddr, "bafybmi") || len(cmd.receiverAddr) < 59 {
+		cmd.log.Error("Invalid sender or receiver DID")
+		return
+	}
+	if cmd.rbtAmount == 0.0 || cmd.rbtAmount < 0.00001 {
```

Comment:
`cmd.rbtAmount < 0.00001` will stop any value from 0.0 & less . So the check  `cmd.rbtAmount == 0.0` is redundant